.github/workflows/validate_pr.yml

```diff
@@ -21,6 +21,8 @@ jobs:
             test
             ci
             chore
-          validateSingleCommit: true
```

Comment:
The removal of this line is important. For context, read [this](https://github.com/amannn/action-semantic-pull-request#legacy-configuration).

I think it's most intuitive, if we're going to enforce token usage, to squash to the PR title. It's worth noting that if additional context is needed when searching through commit history, it could still be viewed on Github. If we _don't_ squash to the PR title, we _could_ set the semantic versioning action to `search_commit_body` via its [inputs](https://github.com/marketplace/actions/git-semantic-version#usage), but it feels like that's introducing a fair bit of complexity that could be annoying to debug if we run into release issues.